_episodes_rmd/07-plot-ggplot2.Rmd

```diff
@@ -114,7 +114,7 @@ This time, we will use the `geom_col()` function as our geometry.
 We will plot countries on the x-axis (listed in alphabetic order
 by default) and gdp per capita on the y-axis.
 
-```{r hist-subset-gapminder}
+```{r hist-subset-gapminder fig.cap = "Barplot of GDP per capita. Country names on x-axis overlap and are not readable"}
```

Comment:
```suggestion
```{r hist-subset-gapminder, fig.cap = "Barplot of GDP per capita. Country names on x-axis overlap and are not readable"}
```